plugins/source/hackernews/resources/plugin/plugin.go

```diff
@@ -0,0 +1,24 @@
+package plugin
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/hackernews/client"
+	"github.com/cloudquery/cloudquery/plugins/source/hackernews/resources/services/items"
+	"github.com/cloudquery/plugin-sdk/plugins/source"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+var (
+	Version = "development"
+)
+
+func Plugin() *source.Plugin {
+	tables := []*schema.Table{
+		items.Items(),
+	}
+	return source.NewPlugin(
+		"hackernews",
+		Version,
+		tables,
```

Comment:
```suggestion
		[]*schema.Table{
			items.Items(),
		},
```
Why not just make it inline?

Reply:
I considered it, but decided it's easier to tell someone in CONTRIBUTING.md to "add new tables to the `tables` variable in plugin.go" than to tell them "add new tables to the `[]*schema.Table` slice (third parameter of `NewPlugin`) in plugin.go". That was my reasoning for not inlining it.